composer.json

```diff
@@ -40,5 +40,15 @@
         "branch-alias": {
             "dev-main": "1.x-dev"
         }
+    },
+    "scripts": {
+        "lint": [
+            "Composer\\Config::disableProcessTimeout",
+            "vendor/bin/phpstan"
+        ],
+        "test": [
+            "Composer\\Config::disableProcessTimeout",
+            "vendor/bin/phpunit"
```

Comment:
There are two main reasons to do this:
1. less things to maintain - We won't have mention the tools we're using all over the place; just tell contributors to run the composer script.
2. it follows some well known or at least, common conventions, that e.g. static code analysis is typically handled by a "lint" task in the project "task manager".

Reply:
I'm still not convinced because composer is quite bad as a task executor. It is not really meant to be a generic task executor

Reply:
@stof could you back up that claim? It might not be _ideal_, but as far as I know, it works quite well.

Besides, anyone could copy those commands and run them in their console, if they really want to. 
It's still better than duplicating this all over the place (github action: `php vendor/bin/phpunit yadda yadda`, contributing.md: "Run `php vendor/bin .....` to run tests...if you're on windows run ....." - repeated for each tool)